content/1-introduction-2-review.md

```diff
@@ -0,0 +1,636 @@
+He thus proposes the Theory of Utterance Selection, which takes linguemes (i.e. the linguistic structure of sounds, words, constructions and utterances) to be replicators, but always part of a larger cycle;
+language speakers are the interactors [-@croft_evolutionary_2013, p. 16], and the theory defines language as the "population of utterances in a speech community" [-@croft_evolutionary_2013, p. 35].
+
+A number of existing studies fit well in this framework.
+@tamariz_cultural_2014, for instance, used a common setup requiring participants to develop a pictorial vocabulary for a pre-given set of words, and modelled the trends participants exhibit in adopting new signs as they go through the interactions of the experiment.
+The authors found that participants do not select new signs neutrally;
+rather, they tend to favour signs they have used in the past, even if their partners use different ones, unless they encounter a sign they find obviously superior in representative power.
+Similarly, in a picture-description transmission chain using an artificial minimal language, @smith_eliminating_2010 showed that the accumulation of individual participant biases will regularise the marking of plurals in the evolved language.
+@kirby_compression_2015, while studying the emergence of structure as the result of combined pressures of expressivity and compressibility [the latter often attributed to learning, @tamariz_culture:_2015], hint to the fact that the way structure emerges and evolves is highly dependent on the combination of such pressures.
+They note, in particular, that "there is some suggestive evidence that structure in language can be modulated by the composition of populations" [@kirby_compression_2015, p. 99]:
+different communication patterns at the population level, or a different fabric in the population responsible for the transmission and evolution of a language (e.g. more second-language learners, or more children learners), should lead to differences in the evolution of language structure.
+Regarding symbolism, @caldwell_cultural_2012 extended the micro-society Pictionary-like task studied by @fay_interactive_2010 to one where participants were gradually replaced, inducing increased symbolism and successful transmission of the evolved symbols at the same time.
+Initial members of the micro-society constructed highly iconic representations of the meanings to convey, but as the experiment introduced newcomers to those signs through observation and consequent use, the drawings gradually lost the iconic link to their referent and became simpler.
+
+Large portions of the iterated learning literature draws on and contributes to a parallel theoretical track which laid down the first analytical predictions for models of Bayesian agents learning and producing languages in chains.
```

Comment:
L'évolution du langage par agents bayésiens est quand même d'un autre niveau (ou d'une autre ambition) que les débuts de modèle de Claidière & Sperber, qui fournissent plutôt des billes au débat mais pas encore de formalisme général utilisable (à la limite les Evolutionary Causal Matrices de Claidière & Sperber 2014, mais il n'y a que ça).

TODO
- [ ] make a subsection out of the analytical literature on language evolution